exec/Chainweb/Lookups.hs

```diff
@@ -205,6 +209,20 @@ mkBlockEvents height cid blockhash pl =  cbes ++ concatMap snd txes
   where
     (cbes, txes) = mkBlockEvents' height cid blockhash pl
 
+mkTransactionSigners :: CW.Transaction -> [Signer]
+mkTransactionSigners t = zipWith3 mkSigner signers sigs [0..]
+  where
+    signers = _pactCommand_signers $ CW._transaction_cmd t
+    sigs = CW._transaction_sigs t
+    mkSigner signer sig idx = Signer
+      (DbHash $ hashB64U $ CW._transaction_hash t)
```

Comment:
Confirming this is the correct unpadded value?

Reply:
Yes.  I fixed that upstream so we wouldn't have to go around and change all the call sites. https://github.com/kadena-io/chainweb-api/blob/master/lib/Chainweb/Api/Hash.hs#L27